libs/design-system-components/src/components/ExtensionEditStep3Form/Collaborators/index.tsx

```diff
@@ -63,7 +63,7 @@ export const Collaborators: React.FC<CollaboratorsProps> = ({
   };
 
   const handleRemoveContributor = (profile: AkashaProfile) => {
-    setAddedContributors(prev => prev.filter(addedProfile => profile.id !== addedProfile?.id));
+    setAddedContributors(prev => prev.filter(addedProfile => profile?.id !== addedProfile?.id));
```

Comment:
this one is not optional

Reply:
or if it's optional, couldn't this lead to false positive?

Reply:
I made it optional on the param. From what I see in the code there is no guarantee the profile param will not be null.

Reply:
I thought more about this, yeah I have scraped the change. It doesn't actually make sense to make the param optional here.